frontend/src/app/coworking/widgets/room-reservation-table/room-reservation-table.widget.html

```diff
@@ -9,7 +9,10 @@
       </tr>
       @for (record of reservationsMap | keyvalue; track record.key) {
       <tr>
-        <th>{{ record.key }}</th>
+        <th>{{ record.key }}
+          <br>{{"Capacity: "}}{{ capacityMap[record.key]}}
+          <br>{{"Room Type: "}}{{ capacityMap[record.key] === 2 ? "Pairing Room" : capacityMap[record.key] < 6 ? "Small Group" : "Large Group"}}
```

Comment:
I recommend removing the `Room Type` label to reduce the width of the column! On mobile devices, this would take up quite a bit of space. 

Optionally, you can also consider changing the capacity label to something like : `# seats`